script/main.py

```diff
@@ -4,23 +4,27 @@
 import re
 
 
-def fetch_arguments():
+def init_argparser():
   """Fetch arguments from command line"""
 
   parser = argparse.ArgumentParser()
-  parser.add_argument('--command', help='entry command')
-  parser.add_argument('--parameter', help='entry parameter: female, male, number or date')
-  args = parser.parse_args()  
-  return (args.command, args.parameter)
+  subparsers = parser.add_subparsers(help='sub-command help')
+  # create the parser for the first command
+  parser_command = subparsers.add_parser('command', help='entry command')
+  # add parameter
+  parser_command.add_argument('--parameter', help='Entry parameter')
+  args = parser.parse_args()
+  return args
+
 
 def init_db():
+  init_argparser()
   '''Initializing the database'''
   
   print('Nothing is implemented')
 
 
 def main():
-  fetch_arguments()
```

Comment:
Call `init_argparser` function here and print the name of the command which was provided as a command line.